package.json

```diff
@@ -28,6 +28,7 @@
     "express": "4.17.1",
     "express-fileupload": "1.2.1",
     "http-errors": "1.8.0",
+    "lodash.merge": "4.6.2",
```

Comment:
Perhaps Ramda is faster than Lodash:
https://www.measurethat.net/Benchmarks/Show/15582/0/deep-merge-lodash-41221-vs-ramda-0271

Reply:
fixed in d17d42b